src/lib/config/common.js

```diff
@@ -2,6 +2,8 @@
 
 export const DEFAULT_CURRENCY = 'CHF';
 
+export const DEFAULT_LANGUAGES = ['ENG'];
```

Comment:
I would go for singular, because it is very misleading otherwise... WDYT?
```suggestion
export const DEFAULT_LANGUAGE = 'ENG';
```

Reply:
yes also we might make a mistake and assign a string instead of the list